packages/aws-lambda-otel-extension/opt/otel-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@opentelemetry/api": "~1.0.4",
+    "@opentelemetry/api": "~1.1.0",
```

Comment:
Yeah I had to bump this because when I would run `npm install` it would throw an error about a version mismatch. I just bumped the `@opentelemetry/core` to `^1.1.0` as well 😎